backend/plugin/parser/utils.go

```diff
@@ -80,7 +113,7 @@ func ExtractTiDBUnsupportStmts(stmts string) ([]string, string, error) {
 	var unsupportStmts []string
 	var supportedStmts bytes.Buffer
 	// We use our bb tokenizer to help us split the multi-statements into statement list.
-	singleSQLs, err := SplitMultiSQL(MySQL, stmts)
+	singleSQLs, err := SplitMultiSQL(MySQL, stmts, true /* filterEmptyStatement */)
```

Comment:
Any reasons that we should not filter empty statements?

I mean let's just remove this argument and make it true.

Reply:
Maybe we need the last comment?
e.g.
```
SELECT * FROM t;
/* comments with some information */
```